### PR TITLE
Feature/dept manager

### PR DESCRIPTION
**Pull request by GeorgeQuantAnalyst**

### CODE REVIEW

File: employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/EmployeeDb.java [hunk 1]
```diff
@@ -5,9 +5,10 @@
 
 import java.time.LocalDate;
 import java.util.Objects;
+import java.util.Set;
 
-@Entity
-@Table(name = "Employees")
+@Entity(name="employees")
```
Comment (LucyQuantAnalyst) [inline employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/EmployeeDb.java:10]: Chybi mezery okolo "="

napr.:
@ Entity(name = "Employee")

File: employee-backoffice-be/src/test/java/com/example/employeebackofficebe/repository/DeptManagerRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+@SpringBootTest
+class DeptManagerRepositoryTest {
+
+    @Autowired
+    private DeptManagerRepository deptManagerRepository;
+
+    @Test
+    void testFindByDepartmentNo() {
+        List<DeptManagerDb> deptManagers = deptManagerRepository.findByDeptNo("d001");
```
Comment (LucyQuantAnalyst) [inline employee-backoffice-be/src/test/java/com/example/employeebackofficebe/repository/DeptManagerRepositoryTest.java:23]: Mozna by byla lepsi konstanta z depNo pro oba testy

File: employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/DeptManagerDb.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
+package com.example.employeebackofficebe.model.db;
+
+import jakarta.persistence.*;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+@Entity(name="DeptManager")
```
Comment (LucyQuantAnalyst) [inline employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/DeptManagerDb.java:8]: Chybi mezery okolo "="

File: employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/EmployeeDb.java (hunk 1 above)
Comment (GeorgeQuantAnalyst) [reply employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/EmployeeDb.java]: fixed

File: employee-backoffice-be/src/test/java/com/example/employeebackofficebe/repository/DeptManagerRepositoryTest.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        Set<Integer> actualManagerIds = deptManagers.stream().map(dm -> dm.getManager().getEmpNo()).collect(Collectors.toSet());
+        Set<Integer> expectedManagerIds = Set.of(110022, 110039);
```
[the remark this answers is not in the document]
Comment (GeorgeQuantAnalyst) [reply employee-backoffice-be/src/test/java/com/example/employeebackofficebe/repository/DeptManagerRepositoryTest.java:25]: Zde to neni nutne.

File: employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/DeptManagerDb.java (hunk 3 above)
Comment (GeorgeQuantAnalyst) [reply employee-backoffice-be/src/main/java/com/example/employeebackofficebe/model/db/DeptManagerDb.java]: fixed

File: employee-backoffice-be/src/test/java/com/example/employeebackofficebe/repository/DeptManagerRepositoryTest.java (hunk 2 above)
Comment (GeorgeQuantAnalyst) [reply employee-backoffice-be/src/test/java/com/example/employeebackofficebe/repository/DeptManagerRepositoryTest.java:23]: Zde to neni nutne.

**Pull request merged**